uber/config.py

```diff
@@ -184,6 +184,11 @@ def DEALER_APPS(self):
         with sa.Session() as session:
             return session.query(sa.Group).filter(sa.Group.tables > 0, sa.Group.cost > 0, sa.Group.status == self.UNAPPROVED).count()
 
+    @request_cached_property
+    @dynamic
+    def ATTENDEE_BADGES_SOLD(self):
```

Comment:
Whoops, I wrote a comment here and forgot to include it in the review! I was just explaining that we actually have these properties magically defined for us via getattr -- you can make up any variable ending in _COUNT and it will attempt to call `get_badge_count_by_type` on the first part of that variable. So if you wrote out c.GUEST_BADGE_COUNT, without defining it separately, you'd get `get_badge_count_by_type(c.GUEST_BADGE)`.

However, that doesn't give us the `request_cached_property` decorator or its benefits.